### ISSUE TITLE
Relocate/Rename individual files and continue seeding

### ISSUE THREAD

**Issue by sand32** [feross/webtorrent#853]
No issue, just a question.  And this may be based in my ignorance of torrents in general, but:

For the application I'm building, it'd be preferable to rename files after downloading, but continue to seed them.  I know that in some graphical BitTorrent clients it's possible to rename a file and continue seeding it, but I'm not sure how I'd go about specifying changed filenames via webtorrent.  I see the option for setting the download directory, but that's not quite the same thing.  Is there some magic sauce I'm missing here?  Would webtorrent search that directory for content-identical files rather than searching by filename?

**Comment by DiegoRBaquero** [comment]
WebTorrent can't do that at the moment, but it'd be a nice feature.

**Comment by sand32** [comment]
Fair enough, I'll have to think on how I want to move forward here.  Thanks for the clarification.
